invenio_app_ils/circulation/views.py

```diff
@@ -117,3 +132,18 @@ def post(self, **kwargs):
         return self.make_response(
             pid, loan, 202, links_factory=self.links_factory
         )
+
+
+class LoanMailResource(IlsCirculationResource):
+    """Loan send email."""
+
+    view_name = "{0}_email"
+
+    @need_permissions('circulation-loan-email')
+    @pass_record
+    def post(self, pid, record, **kwargs):
+        """Loan email post method."""
+        send_overdue_mail(record)
```

Comment:
I think it is better to check here if the loan is really overdue, otherwise return a message that `it is not overdue`, just to be safe.